cobigen-setup/files/cliExecutionCommands.md

```diff
@@ -0,0 +1,18 @@
+CobiGen CLI is installed inside your devonfw distribution. In order to execute it follow the next steps:
+
+1. Run console.bat, this will open a console. Here we will go inside the devonfw ide installation to execute the commands.
+
+`cd root/devonfw`{{execute T1}}

```

Comment:
At this moment you are in the root directory. So you only have to navigate to 'devonfw'.